packages/viewer/src/components/viewer-box-query-tool/viewer-box-query-tool.tsx

```diff
@@ -11,6 +11,8 @@ import {
 
 export type VolumeIntersectionQueryType = 'select' | 'deselect';
 
+export type VolumeIntersectionQueryMode = 'exclusive' | 'inclusive';
```

Comment:
Should we consider adding a third option, `both`? Rather than having `undefined` function as both?

Reply:
Yeah, I thought about having a `both`, but it felt like that didn't communicate enough about the types that would be present. Though the same argument could probably be made for `undefined`. I also considered `exclusive-and-inclusive`, but that felt a bit wordy. Open to ideas here 🙂

Reply:
What would `both` do?

As defined right now:
`inclusive == In | Intersects`
`exclusive == In & ~(Intersects | Out)`

Reply:
@mikemccarty-vertex the `both` would just refer to what queries the tool itself is capable of. I.e. with `both` as the query mode, dragging left-to-right would perform an `exclusive` query and right-to-left would perform an `inclusive` query. With this change, the `inclusive` and `exclusive` modes mean that no matter which way you drag, you'll get that type of query